### PR TITLE
V2

### PR DESCRIPTION
**Pull request by AlvesBernardo**
Cricitcal merge issues fixes. 
Pop Up animation
Pop up stlyle

### CODE REVIEW

File: app/src/main/java/com/example/tileandbackground/MainActivity.kt [hunk 1]
```diff
@@ -1,39 +1,68 @@
...
+        setContentView(R.layout.activity_main)
 
         //get reference to Image
         val arrow = findViewById(R.id.arrowChangePage) as ImageView
         //click Listener
         arrow.setOnClickListener{
             //what should it do
             setContentView(R.layout.activity_rum_coke)
-
         }
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/tileandbackground/MainActivity.kt]: delete empty lines

File: app/src/main/java/com/example/tileandbackground/MainActivity.kt [hunk 2]
```diff
@@ -1,39 +1,68 @@
...
+        showPopUp.setOnClickListener{
+            showPop()
+
+        //get reference to Image
+        val arrow = findViewById(R.id.arrowChangePage) as ImageView
+        //click Listener
+        arrow.setOnClickListener{
+            //what should it do
+            setContentView(R.layout.activity_main)
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/tileandbackground/MainActivity.kt]: delete empty line

File: app/src/main/java/com/example/tileandbackground/MainActivity.kt [hunk 3]
```diff
@@ -1,39 +1,68 @@
...
+        }
+
     }
+    }
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/tileandbackground/MainActivity.kt]: indentation is wrong

File: app/src/main/java/com/example/tileandbackground/MainActivity.kt [hunk 4]
```diff
@@ -1,39 +1,68 @@
...
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/tileandbackground/MainActivity.kt]: delete empty line

File: app/src/main/java/com/example/tileandbackground/MainActivity.kt [hunk 5]
```diff
@@ -1,39 +1,68 @@
...
+       val dialog = builder.create()
 
-        val popUp = PopupMenu(this,view)
-        popUp.inflate(R.menu.popup)
+        dialog.show()
 
     }
+
+
 }
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/tileandbackground/MainActivity.kt]: delete one empty line

File: app/src/main/java/com/example/tileandbackground/Rum_coke.kt [hunk 6]
```diff
@@ -2,19 +2,18 @@ package com.example.tileandbackground
...
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_rum_coke)
 
-        //get reference to Image
-        val backToMain = findViewById(R.id.backPage) as ImageView
-        //click Listener
-        backToMain.setOnClickListener{
-            //what should it do
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/tileandbackground/Rum_coke.kt]: delete empty line

File: app/src/main/res/layout/activity_main.xml [hunk 7]
```diff
@@ -186,18 +144,6 @@
...
-        android:layout_marginEnd="334dp"
-        android:layout_marginBottom="3dp"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toStartOf="@+id/imageView3"
-        app:layout_constraintStart_toEndOf="@+id/imageView6"
->>>>>>> origin/development-app
         app:srcCompat="@drawable/order" />
 
-</androidx.constraintlayout.widget.ConstraintLayout>
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (JoeyKraemer) [inline app/src/main/res/layout/activity_main.xml]: add empty line at the end

File: app/src/main/res/layout/pop_up.xml [hunk 8]
```diff
@@ -59,4 +58,4 @@
         android:layout_marginTop="-470dp"
         app:srcCompat="@drawable/mocktail" />
 
-</LinearLayout>
+</LinearLayout>
```
Comment (JoeyKraemer) [inline app/src/main/res/layout/pop_up.xml]: add empty line at the end

**Review by JoeyKraemer — CHANGES_REQUESTED** [review]

File: app/build.gradle [hunk 9]
```diff
@@ -59,11 +59,15 @@ dependencies {
...
     androidTestImplementation 'androidx.test.ext:junit:1.1.5'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
     androidTestImplementation platform('androidx.compose:compose-bom:2022.10.00')
     androidTestImplementation 'androidx.compose.ui:ui-test-junit4'
     debugImplementation 'androidx.compose.ui:ui-tooling'
     debugImplementation 'androidx.compose.ui:ui-test-manifest'
+    implementation 'androidx.recyclerview:recyclerview:1.2.0'
+    // for adding cardview
+    implementation 'androidx.cardview:cardview:1.0.0'
 }
```
Comment (JoeyKraemer) [inline app/build.gradle]: empty line

File: app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+import android.os.PersistableBundle
+import android.util.Log
+import android.widget.Button
+import android.widget.ImageView
+import androidx.appcompat.app.AppCompatActivity
+import kotlin.random.Random
+
+class AdminActivity : AppCompatActivity() {
+    val url = "http://192.168.0.102:5000/"
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt]: two empty lines

File: app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+
+    // size of graph in dp
+    val graphHeight = 400
+
+    private val graphBarLengths = ArrayList<Int>(4)
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt]: rm empty line

File: app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+    private val graphBars = ArrayList<ImageView>(4)
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        //setContentView(R.layout.)
+
+        /*
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt:21]: delete commented code

File: app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+    val graphHeight = 400
+
+    private val graphBarLengths = ArrayList<Int>(4)
+
+    private val graphBars = ArrayList<ImageView>(4)
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        //setContentView(R.layout.)
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt:20]: commented code

File: app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+
+        /*
+        // get buttons
+        val rebootPiButton = findViewById<Button>(R.id.rebootPiButton)
+        val rebootMachineButton = findViewById<Button>(R.id.rebootMachineButton)
+        val calibrateButton = findViewById<Button>(R.id.calibrateButton)
+         */
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt:27]: empty line

File: app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt [hunk 15]
```diff
@@ -0,0 +1,101 @@
...
+
+        // dynamically get graph bar images
+        for(i in graphBars.indices){
+            graphBars[i] = findViewById<ImageView>(resources.getIdentifier("graphBar$i","id", getPackageName()))
+        }
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt:32]: rm ln

File: app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt [hunk 16]
```diff
@@ -0,0 +1,101 @@
...
+        // import array of amount of drinks sold from main, if it fails, all values are 0
+        val importedArray = intent.getIntArrayExtra("drinkValues")
+        if (importedArray != null) {
+            for(i in graphBarLengths.indices){
+                graphBarLengths[i] = importedArray[i]
+            }
+        }
+
+        /*
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt:40]: rm comments

File: app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt [hunk 17]
```diff
@@ -0,0 +1,101 @@
...
+        if (highest == 0){
+            return
+        }
+        var sizes = calcScaled(graphHeight, highest, graphBarLengths)
+
+        for(i in sizes.indices){
+            graphBars[i].layoutParams.height = sizes[i]
+            // find a way to update the view if it doesn't.
+        }
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt]: rm ln

File: app/src/main/java/com/example/ProjectDrinkMaster/CustomAdapter.kt [hunk 18]
```diff
@@ -0,0 +1,50 @@
...
+import androidx.compose.ui.layout.Layout
+import androidx.recyclerview.widget.RecyclerView.ViewHolder
+
+class CustomAdapter( private val mList: List<ItemsViewModel>) : RecyclerView.Adapter<CustomAdapter.ViewHolder>() {
+    //inner class that is created inside another clas
+    class ViewHolder(view : View) : RecyclerView.ViewHolder(view){
+        val description : TextView = itemView.findViewById(R.id.recyclerViewText)
+        val image : ImageView = itemView.findViewById(R.id.recyclerViewImage)
+        val title : TextView =  itemView.findViewById(R.id.ginTonicText)
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/CustomAdapter.kt]: rm ln

File: app/src/main/java/com/example/ProjectDrinkMaster/CustomAdapter.kt [hunk 19]
```diff
@@ -0,0 +1,50 @@
...
+        val view = LayoutInflater.from(parent.context).inflate(R.layout.recycler_view,parent,false)
+
+        return ViewHolder(view)
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+
+        val ItemsViewModel = mList[position]
+
+        // sets the image to the imageview from our itemHolder class
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/CustomAdapter.kt:29]: indentation

File: app/src/main/java/com/example/ProjectDrinkMaster/CustomAdapter.kt [hunk 20]
```diff
@@ -0,0 +1,50 @@
...
+        holder.description.text = ItemsViewModel.text
+       // holder.image. = ItemsViewModel.getImage()
+        holder.image.setImageResource(ItemsViewModel.image)
+        holder.title.text = ItemsViewModel.title
+
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/CustomAdapter.kt]: rm ln

File: app/src/main/java/com/example/ProjectDrinkMaster/CustomAdapter.kt [hunk 21]
```diff
@@ -0,0 +1,50 @@
...
+        return ViewHolder(view)
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+
+        val ItemsViewModel = mList[position]
+
+        // sets the image to the imageview from our itemHolder class
+        holder.description.text = ItemsViewModel.text
+       // holder.image. = ItemsViewModel.getImage()
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/CustomAdapter.kt:31]: commented code

File: app/src/main/java/com/example/ProjectDrinkMaster/CustomAdapter.kt [hunk 22]
```diff
@@ -0,0 +1,50 @@
...
+
+
+    }
+
+    override fun getItemCount(): Int {
+
+        return mList.size
+    }
+
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/CustomAdapter.kt]: rm ln

File: app/src/main/java/com/example/ProjectDrinkMaster/CustomAdapter.kt [hunk 23]
```diff
@@ -0,0 +1,50 @@
...
+}
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/CustomAdapter.kt:39]: add ln

File: app/src/main/java/com/example/ProjectDrinkMaster/ItemsViewModel.kt [hunk 24]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/ItemsViewModel.kt:1]: rm ln

File: app/src/main/java/com/example/ProjectDrinkMaster/ItemsViewModel.kt [hunk 25]
```diff
@@ -0,0 +1,10 @@
...
+package com.example.ProjectDrinkMaster
+
+import android.media.Image
+
+data class ItemsViewModel(var text: String,  var image: Int, var title: String) {
+
+
+
+}
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/ItemsViewModel.kt:7]: add ln

File: app/src/main/java/com/example/ProjectDrinkMaster/ItemsViewModel.kt [hunk 26]
```diff
@@ -0,0 +1,10 @@
+
+package com.example.ProjectDrinkMaster
+
+import android.media.Image
+
+data class ItemsViewModel(var text: String,  var image: Int, var title: String) {
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/ItemsViewModel.kt]: rm line

File: app/src/main/java/com/example/ProjectDrinkMaster/MainActivity.kt [hunk 27]
```diff
@@ -0,0 +1,125 @@
...
+    private lateinit var newRecyclerView: RecyclerView
+    private lateinit var newArrayList: ArrayList<Rum_coke>
+    lateinit var  heading : Array<String>
+    lateinit var news : Array<Rum_coke>
+    lateinit var imageId : Array<Int>
+    lateinit var text : Array<String>
+    private var drinkList = ArrayList<ItemsViewModel>()
+    private lateinit var drinkAdapter : CustomAdapter
+
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/MainActivity.kt]: rm ln

File: app/src/main/java/com/example/ProjectDrinkMaster/MainActivity.kt [hunk 28]
```diff
@@ -0,0 +1,125 @@
...
+    @SuppressLint("MissingInflatedId")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        //get reference to Image
+        /*
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/MainActivity.kt]: rm commented code

File: app/src/main/java/com/example/ProjectDrinkMaster/MainActivity.kt [hunk 29]
```diff
@@ -0,0 +1,125 @@
...
+
+         */
+
+        var showPopUp = findViewById<ImageButton>(R.id.imageButton)
+        var countMocktail = 0
+        var countRumCoke = 0
+        var countLemonade = 0
+        var countCokeLemon = 0
+
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/MainActivity.kt]: rm ln

File: app/src/main/java/com/example/ProjectDrinkMaster/MainActivity.kt [hunk 30]
```diff
@@ -0,0 +1,125 @@
...
+        var test = LinearLayoutManager(applicationContext)
+        recyclerView.layoutManager = LinearLayoutManager(this,LinearLayoutManager.HORIZONTAL,false)
+        recyclerView.adapter = drinkAdapter
+        prepareDiffernetDrinks()
+
+       showPopUp.setOnClickListener {
+            showPop()
+        }
+
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/MainActivity.kt]: rm ln

File: app/src/main/java/com/example/ProjectDrinkMaster/MainActivity.kt [hunk 31]
```diff
@@ -0,0 +1,125 @@
...
+    }
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/MainActivity.kt:50]: rm line

File: app/src/main/java/com/example/ProjectDrinkMaster/MainActivity.kt [hunk 32]
```diff
@@ -0,0 +1,125 @@
...
+        drinkList.add(drink)
+        drink = ItemsViewModel("Rum and Coke is a timeless cocktail that brings together the smooth, rich flavors of rum with the crisp, effervescence of cola. This simple yet delightful drink has been enjoyed by cocktail enthusiasts for decades.",R.drawable.rum,"Rum cola")
+        drinkList.add(drink)
+        drink = ItemsViewModel("Lemonade is a classic and refreshing beverage that embodies the bright and tangy flavors of fresh lemons. With its simple yet irresistible blend of lemon juice, water, and sweetener, lemonade has been a beloved thirst-quencher for generations.",R.drawable.lemonade,"Lemonde")
+        drinkList.add(drink)
+        drinkAdapter.notifyDataSetChanged()
+
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/MainActivity.kt]: rm ln

File: app/src/main/java/com/example/ProjectDrinkMaster/MainActivity.kt [hunk 33]
```diff
@@ -0,0 +1,125 @@
...
+    }
+
+}
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/MainActivity.kt:104]: rm ln

File: app/src/main/java/com/example/ProjectDrinkMaster/readRequest.kt [hunk 34]
```diff
@@ -0,0 +1,18 @@
...
+    private val page = page
+    private var result = ""
+    override fun doInBackground(vararg params: Void?): String {
+        result = URL("$url$page").readText()
+        return result
+    }
+    fun getResult(): String{
+        return result
+    }
+}
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/readRequest.kt:18]: add ln

File: app/src/main/java/com/example/ProjectDrinkMaster/sendRequest.kt [hunk 35]
```diff
@@ -0,0 +1,26 @@
...
+
+        with(url.openConnection() as HttpURLConnection){
+            requestMethod = "GET"
+            Log.d("networkRequest","\nSent 'GET' request to URL : $url; Response Code : $responseCode")
+
+        }
+        return "done"
+    }
+
+}
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/sendRequest.kt:23]: add ln

File: app/src/main/java/com/example/ProjectDrinkMaster/sendRequest.kt [hunk 36]
```diff
@@ -0,0 +1,26 @@
...
+class sendRequest (url: String, key: String, value: String) : AsyncTask<Void, Void, String>() {
+    val url = URL("$url?$key=$value")
+
+    @Deprecated("Deprecated in Java")
+    override fun doInBackground(vararg params: Void?): String {
+
+        with(url.openConnection() as HttpURLConnection){
+            requestMethod = "GET"
+            Log.d("networkRequest","\nSent 'GET' request to URL : $url; Response Code : $responseCode")
+
```
Comment (JoeyKraemer) [inline app/src/main/java/com/example/ProjectDrinkMaster/sendRequest.kt]: rm line

File: app/src/main/res/layout/activity_main.xml (hunk 7 above)
Comment (JoeyKraemer) [reply app/src/main/res/layout/activity_main.xml]: still not changed

File: app/src/main/res/layout/activity_rum_coke.xml [hunk 37]
```diff
@@ -7,159 +7,159 @@
...
+        android:layout_marginBottom="342dp"
+        android:clickable="true"
+        android:rotation="180"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/arrowChangePageRigth"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/title"
+        app:srcCompat="@drawable/right_arrow" />
 
 </androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (JoeyKraemer) [inline app/src/main/res/layout/activity_rum_coke.xml:165]: add ln

File: app/src/main/res/layout/gin_tonic.xml [hunk 38]
```diff
@@ -0,0 +1,150 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="400dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/imageView6"
+        app:layout_constraintTop_toBottomOf="@+id/title"
+        app:srcCompat="@drawable/order" />
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (JoeyKraemer) [inline app/src/main/res/layout/gin_tonic.xml:149]: add ln

File: app/src/main/res/layout/gintonic.xml [hunk 39]
```diff
@@ -0,0 +1,151 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="400dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/imageView6"
+        app:layout_constraintTop_toBottomOf="@+id/title"
+        app:srcCompat="@drawable/order" />
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (JoeyKraemer) [inline app/src/main/res/layout/gintonic.xml:151]: add ln

File: app/src/main/res/layout/layout_recycler.xml [hunk 40]
```diff
@@ -0,0 +1,22 @@
...
+        app:layout_constraintTop_toTopOf="parent"
+        tools:layout_editor_absoluteX="-9dp">
+            <ImageView
+                android:id="@+id/imageView9"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_weight="1"
+                tools:srcCompat="@tools:sample/avatars" />
+    </ScrollView>
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (JoeyKraemer) [inline app/src/main/res/layout/layout_recycler.xml]: add ln

File: app/src/main/res/layout/ok_box.xml [hunk 41]
```diff
@@ -29,20 +33,16 @@
         android:fontFamily="@font/share_regular"
         android:text="Your drink will be prepared shortly"
         android:textColor="@color/white"
+
```
Comment (JoeyKraemer) [inline app/src/main/res/layout/ok_box.xml]: rm ln

File: app/src/main/res/layout/ok_box.xml [hunk 42]
```diff
@@ -29,20 +33,16 @@
...
-        android:layout_marginTop="270dp"
-        app:srcCompat="@drawable/lemon" />
+        android:id="@+id/lemon"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="630dp"
+        android:scaleType="fitXY"
+        app:srcCompat="@drawable/lemon_small" />
+
 </LinearLayout>
```
Comment (JoeyKraemer) [inline app/src/main/res/layout/ok_box.xml:47]: add ln

File: app/src/main/res/layout/pop_up.xml [hunk 43]
```diff
@@ -1,62 +1,67 @@
...
 
-    <ImageView
-        android:id="@+id/mocktail"
-        android:layout_width="98dp"
-        android:layout_height="112dp"
-        android:layout_marginTop="-470dp"
-        app:srcCompat="@drawable/mocktail" />
 
-</LinearLayout>
+</LinearLayout>
```
Comment (JoeyKraemer) [inline app/src/main/res/layout/pop_up.xml]: add ln

File: app/src/main/res/layout/recycler_view.xml [hunk 44]
```diff
@@ -0,0 +1,48 @@
...
+    android:layout_height="match_parent"
+    android:background="@color/ligthBeige"
+    app:cardElevation="6dp">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+
```
Comment (JoeyKraemer) [inline app/src/main/res/layout/recycler_view.xml]: rm ln

File: app/src/main/res/layout/recycler_view.xml [hunk 45]
```diff
@@ -0,0 +1,48 @@
...
+                android:text="TextView" />
+
+            <ImageView
+                android:id="@+id/recyclerViewImage"
+                android:layout_width="675dp"
+                android:layout_height="match_parent"
+                android:layout_marginLeft="600dp"
+                app:srcCompat="@drawable/rum" />
+
+        </FrameLayout>
```
Comment (JoeyKraemer) [inline app/src/main/res/layout/recycler_view.xml:37]: rm ln

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 46]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@drawable/ic_launcher_background" />
-    <foreground android:drawable="@drawable/ic_launcher_foreground" />
-    <monochrome android:drawable="@drawable/ic_launcher_foreground" />
+    <background android:drawable="@color/ic_launcher_background"/>
+    <foreground android:drawable="@mipmap/ic_launcher_foreground"/>
 </adaptive-icon>
```
Comment (JoeyKraemer) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml:5]: add ln

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml [hunk 47]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@drawable/ic_launcher_background" />
-    <foreground android:drawable="@drawable/ic_launcher_foreground" />
-    <monochrome android:drawable="@drawable/ic_launcher_foreground" />
+    <background android:drawable="@color/ic_launcher_background"/>
+    <foreground android:drawable="@mipmap/ic_launcher_foreground"/>
 </adaptive-icon>
```
Comment (JoeyKraemer) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml:5]: add ln

File: app/src/main/res/values/ic_launcher_background.xml [hunk 48]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="ic_launcher_background">#161616</color>
+</resources>
```
Comment (JoeyKraemer) [inline app/src/main/res/values/ic_launcher_background.xml:4]: add ln

File: app/src/main/res/values/strings.xml [hunk 49]
```diff
@@ -1,4 +1,11 @@
...
-    <string name="app_name">Tile and background</string>
+    <string name="app_name">Project DrinkMaster</string>
     <string name="title_activity_ok_box">ok_box</string>
+    <string-array name="tilteOptions">
+        <item>Gin n Tonic</item>
+        <item>Lemonade</item>
+        <item>Rum cola</item>
+        <item>Cola lemonade</item>
+    </string-array>
+
```
Comment (JoeyKraemer) [inline app/src/main/res/values/strings.xml:10]: add ln

File: app/src/main/res/values/themes.xml [hunk 50]
```diff
@@ -1,9 +1,9 @@
...
     <!-- Base application theme. -->
-    <style name="Base.Theme.TileAndBackground" parent="Theme.Material3.DayNight.NoActionBar">
+    <style name="Base.Theme.ProjectDrinkMaster" parent="Theme.Material3.DayNight.NoActionBar">
         <!-- Customize your light theme here. -->
         <!-- <item name="colorPrimary">@color/my_light_primary</item> -->
     </style>
 
-    <style name="Theme.TileAndBackground" parent="Base.Theme.TileAndBackground" />
+    <style name="Theme.ProjectDrinkMaster" parent="Base.Theme.ProjectDrinkMaster" />
 </resources>
```
Comment (JoeyKraemer) [inline app/src/main/res/values/themes.xml:9]: add ln

File: build.gradle [hunk 51]
```diff
@@ -1,6 +1,6 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id 'com.android.application' version '8.0.1' apply false
-    id 'com.android.library' version '8.0.1' apply false
+    id 'com.android.application' version '8.0.2' apply false
+    id 'com.android.library' version '8.0.2' apply false
     id 'org.jetbrains.kotlin.android' version '1.8.0' apply false
 }
```
Comment (JoeyKraemer) [inline build.gradle]: add ln

**Review by JoeyKraemer — CHANGES_REQUESTED** [review]
Please fix the pointed out styling mistakes

File: app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt (hunk 13 above)
Comment (AlvesBernardo) [reply app/src/main/java/com/example/ProjectDrinkMaster/AdminActivity.kt:20]: Yohan will need them for later

**Review by JoeyKraemer — APPROVED** [review]

**Pull request merged**